openkfw/TruBudget
# #1901 workflow item funding source

### Checklist




### Description

* added option to select workflow item funding organization

### How to test

1. run app, go to any project and subproject that has a specified budget
2. try creating new workflow item with allocated or paid budget
3. there should be listed funding sources in the dropdown
4. select one and define amount, submit workflow item
5. view newly created WF item and budget source should be defined
6. edit WF item, try different scenarios like changing funding source, removing budget allocation option by selecting "not applicable" in radio buttons and so on...
7. WF item should be updated successfully 




Closes #1901
